x-pack/plugins/security_solution/server/types.ts

```diff
@@ -39,6 +39,7 @@ export interface SecuritySolutionApiRequestHandlerContext {
   getSpaceId: () => string;
   getRuleDataService: () => IRuleDataService;
   getRuleExecutionLog: () => IRuleExecutionLogForRoutes;
+  getRacClient: () => (req: KibanaRequest) => Promise<AlertsClient>;
```

Comment:
Why does it return a function instead of just being a function?

```ts
getRacClient: (req: KibanaRequest) => Promise<AlertsClient>;
```

Reply:
Updated, think I was just using an inferred type from somewhere else, but ya this is not needed, thanks changed in https://github.com/elastic/kibana/pull/135340/commits/cde7bd60dbcd7cd083a36a707c92806f96f9c36a